unit-tests-java8/src/test/java/org/eclipse/collections/test/map/MapIterableTestCase.java

```diff
@@ -183,6 +184,18 @@ default void MapIterable_forEachKeyValue()
         assertEquals(this.newMutableForFilter(4, 3, 2), forEachKey);
     }
 
+    @Test
+    default void MapIterable_injectIntoKeyValue()
+    {
+        MapIterable<Integer, Integer> map1 = this.newWithKeysValues(3, 3, 2, 2, 1, 1);
+        Integer sum1 = map1.injectIntoKeyValue(0, (sum, key, value) -> sum + key + value);
+        assertEquals(12, sum1);
+
+        MutableMap<Integer, String> result = map1.injectIntoKeyValue(Maps.mutable.empty(), (map, key, value)
```

Comment:
Please move the lambda parameter list to the next line so it is aligned with the separator and the expression.